src/platform/common/refBool.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
 export class RefBool {
```

Comment:
Surprised how the linters didn't catch this, this was used in the cell execution code.